apps/chat/src/utils/app/folders.ts

```diff
@@ -192,7 +192,10 @@ export const getFolderIdByPath = (path: string, folders: FolderInterface[]) => {
 export const getPathToFolderById = (
   folders: FolderInterface[],
   starterId: string | undefined,
-  prepareNames = false,
+  prepareNames?: {
+    forRenaming?: boolean;
```

Comment:
```suggestion
    forRenaming: boolean;
```

Reply:
I think options could change in feature. By default forRenaming could be undefined